src/main/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -134,6 +134,8 @@
     /** A generic static NON_REF allele for use */
     public final static String NON_REF_STRING = "<NON_REF>";
```

Comment:
Could you comment that these two strings are actually two different representations of the same thing?